services/wallet/router/router_v2.go

```diff
@@ -781,8 +819,8 @@ func (r *Router) resolveCandidates(ctx context.Context, input *RouteInputParams,
 		return nil, nil, errors.CreateErrorResponseFromError(err)
 	}
 
-	appendProcessorErrorFn := func(processorName string, err error) {
-		log.Error("routerv2.resolveCandidates error", "processor", processorName, "err", err)
+	appendProcessorErrorFn := func(processorName string, sendType SendType, fromChainID uint64, toChainID uint64, amount *big.Int, err error) {
```

Comment:
not sure if you still intended to log that extra info, otherwise you can just revert this and all the function calls